arrow/arrow.py

```diff
@@ -1013,28 +1046,28 @@ def humanize(
                 elif diff < 90:
                     return locale.describe("minute", sign, only_distance=only_distance)
                 elif diff < 2700:
-                    minutes = sign * int(max(delta / 60, 2))
+                    minutes = sign * float(max(delta / 60, 2))
```

Comment:
Won't using a float impact our humanize functionality since we are no longer truncating?